src/SFML/Graphics/Font.cpp

```diff
@@ -292,37 +284,34 @@ bool Font::loadFromStream(InputStream& stream)
     }
 
     // Prepare a wrapper for our stream, that we'll pass to FreeType callbacks
-    fontHandles->streamRec                     = std::make_unique<FT_StreamRec>();
```

Comment:
Not sure why this was being allocated separately before -- it looks unnecessary to me unless I am missing something.